wiki/floodgate/index.md

```diff
@@ -1,15 +1,14 @@
 ---
-title: Floodgate
+title: Overview
 permalink: /floodgate/
 description: 'Floodgate is a hybrid mode plugin that allows Minecraft: Bedrock Accounts to join Minecraft: Java Edition servers without needing a Minecraft: Java Edition account.'
 ---
 
-# Floodgate
-## What is Floodgate? {#what-is-floodgate}
-Floodgate is a hybrid mode plugin that allows Minecraft: Bedrock Accounts to join Minecraft: Java Edition servers without needing a Minecraft: Java Edition account. 
-This is something you install in addition to Geyser. Unlike Geyser, Floodgate can only be installed as a plugin on Spigot (including Paper and forks), Bungeecord, Velocity, and as a mod on Fabric servers, and can only be utilized on servers that have it installed.
+# Overview
+Floodgate is a hybrid mode plugin. It introduces new features:
+- being able to join Minecraft: Java Edition servers with Minecraft: Bedrock Accounts without needing a Minecraft: Java Edition account. 
+- being able to [see Bedrock player skins on Java edition](/wiki/floodgate/features#what-is-skin-uploading).
+- being able to [use Bedrock forms](/wiki/geyser/forms/).
```

Comment:
```suggestion
- Plugins being able to [send Bedrock forms](/wiki/geyser/forms/).
```

Reply:
`Ability for plugins to` I think would fit better?